Cargo.toml

```diff
@@ -17,7 +17,7 @@ appveyor = { repository = "serde-rs/json" }
 
 [dependencies]
 serde = "1.0.60"
-linked-hash-map = { version = "0.5", optional = true }
+indexmap = { version = "1.0", optional = true }
```

Comment:
Please change the Travis configuration to add a build at indexmap's minimum supported compiler version, and remove any `preserve_order` builds below that compiler version.

Reply:
Done